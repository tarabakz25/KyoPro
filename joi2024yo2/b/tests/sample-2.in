5 3 3
10 1
40 3
30 2
20 1
50 3
1 2 4
3 3 5
2 1 5
