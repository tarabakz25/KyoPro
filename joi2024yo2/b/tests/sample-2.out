80
75
135
